x-pack/legacy/plugins/uptime/public/hooks/use_index_pattern.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { useEffect, Dispatch } from 'react';
+import { HttpHandler } from 'kibana/public';
+import { useKibana } from '../../../../../../src/plugins/kibana_react/public';
+
+const getIndexPattern = async (fetch: HttpHandler): Promise<any> => {
```

Comment:
Given that this is invoked exactly once from `useIndexPattern` do we really need a separate function here?

Reply:
Addressed in https://github.com/elastic/kibana/pull/53726/commits/6a00da3426b967c7c4367b7b57e7d373decfb8a0.